.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+### Summary of Changes
+
+Your summary goes here!
+
+### PR Checklist
+
+Make sure the following checklist items are done before merging this PR.
+
+- [ ] Update `CHANGELOG.md` in the `## Unreleased` section ([instructions here])
+- [ ] Make sure tests are passing
```

Comment:
Isn't this automated already? Usually if Jenkins build fails (which run tests) then the PR cannot be merged.

Reply:
That's true, we can remove it if we feel like it's just adding noise to the PRs!